File: Rentacar.API/Rentacar.Entities/Make.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Rentacar.Entities
{
    public partial class Make
    {
        public Make()
        {
            Models = new HashSet<Model>();
        }

        public int MakeId { get; set; }
        public string MakeName { get; set; }
        public string MakeDescription { get; set; }

        public virtual ICollection<Model> Models { get; set; }
    }
}

File: rentacar.flutter/lib/models/make.dart
class Make {
  int makeId = 0;
  String makeName = '';
  String makeDescription = '';

  Make.fromJson(Map<String, dynamic> makeMap) {
    makeId = makeMap['makeId'] ?? 0;
    makeName = makeMap['makeName'] ?? '';
    makeDescription = makeMap['makeDescription'] ?? '';
  }
}
